packages/components-js/projects/components-manager-core/package.json

```diff
@@ -4,20 +4,20 @@
   "private": true,
   "license": "UNLICENSED",
   "main": "dist/index.js",
+  "module": "dist/esm/index.js",
   "types": "dist/types/index.d.ts",
   "scripts": {
-    "build": "yarn clean:build && tsc --build tsconfig.json",
+    "build": "yarn clean:build && tsc --build tsconfig.json && tsc --project tsconfig.json --module CommonJS --outDir ./dist",
```

Comment:
Why is the CJS build needed? For testing reasons?

Reply:
I think for jest unit tests.